backend/tracim_backend/lib/utils/translation.py

```diff
@@ -26,7 +26,10 @@ class Translator(object):
     def __init__(self, app_config: 'CFG', default_lang: str = None, fallback_lang: str = None):  # nopep8
         self.config = app_config
         if not fallback_lang:
-            fallback_lang = DEFAULT_FALLBACK_LANG
+            if app_config.DEFAULT_LANG:
+                fallback_lang = app_config.DEFAULT_LANG
+            else:
+                fallback_lang = DEFAULT_FALLBACK_LANG
```

Comment:
It seems useless to manage case where `app_config.DEFAULT_LANG` is not set because this config is always set...

Reply:
done.